eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/AssemblyCheckerProcessor.java

```diff
@@ -57,7 +59,7 @@ private String getSequenceUsingSynonyms(ContigSynonyms contigSynonyms, long star
         if ((sequence = getSequence(contigSynonyms.getUcsc(), start, end)) != null) {
             return sequence;
         }
-        throw new IllegalArgumentException("Contig " + contigSynonyms.getSequenceName() + " not found in FASTA file");
+        throw new IllegalArgumentException("Contig " + contigSynonyms.toString() + " not found in fasta file");
```

Comment:
As mentioned in a previous review, FASTA is actually the correct name. It doesn't need to be changed anywhere else.